app/src/main/java/se/inera/intyg/minaintyg/certificate/CertificateController.java

```diff
@@ -69,4 +79,35 @@ public void sendCertificateToRecipient(@PathVariable String certificateId) {
             .build()
     );
   }
+
+  @PostMapping(value = "/{certificateId}/pdf", produces = "application/pdf")
+  public ResponseEntity printCertificate(
```

Comment:
Sonarlint warns about this raw use of ResponseEntity. Maybe should update this to ResponseEntity<byte[]>